tools/aa_classifier.cwl

```diff
@@ -0,0 +1,46 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: aa-classifier
+doc: "This tools takes output from amplicon architect and classifies them"
+requirements: 
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement 
+  - class: DockerRequirement
+    dockerPull: 'jluebeck/prepareaa:latest'
+  - class: ResourceRequirement
+    ramMin: 4000
+    coresMin: $(inputs.threads)
+  - class: InitialWorkDirRequirement
+    listing: [$(inputs.data_repo)]
+baseCommand: [export]
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      AA_DATA_REPO=$(inputs.data_repo.path)
+      && export AA_SRC=/home/programs/AmpliconArchitect-master/src
```

Comment:
this can be set with: https://www.commonwl.org/v1.2/CommandLineTool.html#EnvVarRequirement

Reply:
I was thinking about that ,but then got too deep. Def would be cleaner, so I will switch and hope this isn't oddly shell-dependent